Sources/Citadel/TTY/Client/TTY.swift

```diff
@@ -304,4 +373,47 @@ extension SSHClient {
         
         return ExecCommandStream(stdout: stdout, stderr: stderr)
     }
+    
+    /// Requests a shell to be invoked and executes a command on the remote server.
+    /// - Parameters:
+    /// - command: The command to execute.
+    public func executeInShell(command: String) async throws -> AsyncThrowingStream<ByteBuffer, Error> {
+        var streamContinuation: AsyncThrowingStream<ByteBuffer, Error>.Continuation!
+        let stream = AsyncThrowingStream<ByteBuffer, Error> { continuation in
+            streamContinuation = continuation
+        }
+        let handler = ShellCommandHandler(logger: logger) { channel, output in
+            switch output {
+            case .shellSuccess:
+                let commandData = SSHChannelData(type: .channel,
```

Comment:
This should only happen once, can we ensure this can't happen more than once?

Reply:
I'll add a `Bool` property that keeps track of it.